.github/actions/cilium-config/action.yml

```diff
@@ -31,6 +31,9 @@ inputs:
   encryption-node:
     description: 'Enable node-to-node encryption (WireGuard only)'
     default: false
+  encryption-overlay:
```

Comment:
@ldelossa This is just some CI-side name. It doesn't need to match the flag name; that happens below in this file. I used `encryption-overlay` because we already have other `encryption` prefixes.

Reply:
SGTM.